packages/components/src/components/tabs/tabs.hbs

```diff
@@ -5,7 +5,7 @@
   LICENSE file in the root directory of this source tree.
 -->
 
```

Comment:
we may need unique IDs for the examples otherwise the tabs will control the content for all examples